### PR TITLE
fix: Delete useless Third party libraries dependencies declaration - Meeds-io/MIPs#42

### PR DESCRIPTION
**Pull request by boubaker**
This change will delete useless Third party libraries versions declarations.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,59 +62,33 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-    <net.sourceforge.cssparser.version>0.9.18</net.sourceforge.cssparser.version>
     <net.sourceforge.nekohtml.version>1.9.22</net.sourceforge.nekohtml.version>
-    <net.sourceforge.htmlcleaner.version>2.7</net.sourceforge.htmlcleaner.version>
-    <org.apache.geronimo.specs.geronimo-stax-api_1.0_spec.version>1.0.1</org.apache.geronimo.specs.geronimo-stax-api_1.0_spec.version>
     <org.apache.httpcomponents.httpclient.version>4.5.13</org.apache.httpcomponents.httpclient.version>
-    <org.apache.httpcomponents.httpcore.version>4.4.5</org.apache.httpcomponents.httpcore.version>
-    <org.apache.jackrabbit.version>1.6.5</org.apache.jackrabbit.version>
-    <org.apache.lucene.version>3.6.2</org.apache.lucene.version>
-    <org.apache.pdfbox.version>2.0.24</org.apache.pdfbox.version>
     <org.apache.poi.version>5.2.2</org.apache.poi.version>
```
Comment (rdenarie) [inline pom.xml:90]: to delete ?

File: pom.xml [hunk 2]
```diff
@@ -1069,102 +643,37 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-      <dependency>
-        <groupId>org.antlr</groupId>
-        <artifactId>antlr</artifactId>
-        <version>${version.org.antlr}</version>
-      </dependency>
       <dependency>
         <groupId>org.antlr</groupId>
         <artifactId>antlr-runtime</artifactId>
         <version>${version.org.antlr}</version>
       </dependency>
```
Comment (rdenarie) [inline pom.xml:671]: to delete ?

File: pom.xml [hunk 3]
```diff
@@ -1069,102 +643,37 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-      <dependency>
-        <groupId>org.antlr</groupId>
-        <artifactId>gunit</artifactId>
-        <version>${version.org.antlr}</version>
-      </dependency>
       <dependency>
         <groupId>antlr</groupId>
         <artifactId>antlr</artifactId>
         <version>${version.antlr}</version>
       </dependency>
```
Comment (rdenarie) [inline pom.xml:676]: to delete ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (boubaker) [reply pom.xml:90]: In fact, it's used by gamification when exporting XLSX. May be should be moved to gamification addon only.

File: pom.xml (hunk 2 above)
Comment (boubaker) [reply pom.xml:671]: In fact, it's used in Meeds packaging.

File: pom.xml (hunk 3 above)
Comment (boubaker) [reply pom.xml:676]: Seems used as third party library and packaged in Meeds.

**Pull request merged**